index.html

```diff
@@ -5,10 +5,130 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="./css/style.css">
-  <title>Digital Marketing Agency</title>
+  <script src="https://kit.fontawesome.com/035535dd6b.js" crossorigin="anonymous"></script>
+  <title>GH Comedy Crusade</title>
 </head>
-<body>
+<body id="body">
+  <!-- Social Navigation -->
+  <nav class="nav">
+    <div class="container">
+      <div class="nav__social">
+        <ul class="nav__link">
+          <li><a href="#" class="nav__link--items"><i class="fa-brands fa-facebook-f"></i></a></li>
+          <li><a href="#" class="nav__link--items"><i class="fa-brands fa-twitter"></i></a></li>
+          <li><a href="#" class="nav__link--items">English</a></li>
+          <li><a href="#" class="nav__link--items">Blog</a></li>
+          <li><a href="#" class="nav__link--items">Logout</a></li>
+        </ul>
+      </div>
+    </div>
+  </nav>
```

Comment:
- [ ] Good job styling your navbar for desktop version :+1:, but as a project requirement you should stick as much as possible to the template, so kindly align the items of the top dark bar to the right :+1: 

**Your design**

![Screenshot from 2023-02-08 06-38-15](https://user-images.githubusercontent.com/105117832/217520391-ed8e138f-4d01-4794-ad71-fbb8e3dd24c3.png)

**Template**

![Screenshot from 2023-02-08 06-39-02](https://user-images.githubusercontent.com/105117832/217520441-448af7e3-3d15-4b76-9aeb-7ac0efa5af35.png)